src/content/documentation/publish/source-code-submission.md

```diff
@@ -63,15 +63,22 @@ To reproduce the build, the reviewer runs the instructions you provided and then
 - details, including required version and installation instructions, of any tools or utilities that need to be downloaded, for example, [yuicompressor](http://yui.github.io/yuicompressor/).
 - a list of all the commands to generate an identical copy of the extension from the source code, for example, npm install or a grunt target. Ideally, you should include every command in the build script file.
 
+Reviewers will try to replicate your build environment if specified. However, if you do not specify the build environment, here's the default build environment that reviewers use:
```

Comment:
Could we add a subheading here? Would be useful for when we include the link to this section in the canned response.

Reply:
This was addressed in commit 6940b3b.